packages/shared/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@chromaui/shared-e2e",
+  "private": true,
+  "version": "0.0.1",
+  "description": "Shared Chromatic E2E",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chromaui/test-archiver.git",
+    "directory": "packages/shared"
+  },
+  "author": "Chromatic <tom@chromatic.com>",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "clean": "rimraf ./dist",
+    "prebuild": "yarn clean",
+    "build": "tsup",
+    "build:watch": "yarn build --watch",
+    "test:unit": "jest",
+    "lint": "eslint src/*",
+    "prettier": "prettier"
+  },
+  "devDependencies": {
+    "@auto-it/released": "^10.32.6",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
+    "@chromaui/archive-storybook": "~0.0.15",
+    "@jest/types": "^27.0.6",
+    "@playwright/test": "^1.39.0",
+    "@storybook/addon-essentials": "^7.5.2",
+    "@storybook/cli": "^7.5.2",
+    "@storybook/eslint-config-storybook": "^3.1.2",
+    "@storybook/server-webpack5": "^7.5.2",
+    "@testing-library/dom": "^8.1.0",
+    "@testing-library/react": "^12.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/express": "^4.17.17",
+    "@types/fs-extra": "^11.0.1",
+    "@types/jest": "^27.0.3",
+    "@types/node": "^16.4.1",
+    "@types/node-fetch": "^2.6.6",
+    "auto": "^10.3.0",
+    "babel-jest": "^27.0.6",
+    "concurrently": "^7.0.0",
+    "cypress": "^13.4.0",
```

Comment:
Since the shared package is test-framework-agnostic, does it work OK if cypress isn't a dependency?

Reply:
I'll try to remove that!